Source/Injector/Hooks/DarkSouls2/DS2_ReplaceServerAddressHook.cpp

```diff
@@ -99,10 +107,22 @@ bool DS2_ReplaceServerAddressHook::PatchKey(Injector& injector)
         {
             // If the memory is not writable yet, modify its protection (steam drm fucks with the protection during boot).
             MEMORY_BASIC_INFORMATION info;
-            if (VirtualQuery((void*)key, &info, sizeof(info)) == 0 ||
-                ((info.Protect & PAGE_READWRITE) == 0 && (info.Protect & PAGE_EXECUTE_READWRITE) == 0))
+            // If the programm is running on wine
+            if(winePrefix != nullptr)
```

Comment:
space after if please, also the comment doesn't really add anything, please remove it.